### PR TITLE
Change base image to 'openjdk:8'

### PR DESCRIPTION
**Pull request by hyangtack**
Modifications:
- Do not use 'java:openjdk-8-jre' anymore which is deprecated
- Copy missing binaries to the docker image

### CODE REVIEW

File: dist/Dockerfile [hunk 1]
```diff
@@ -1,29 +1,30 @@
 #
 # CentralDogma Dockerfile
 #
-FROM java:openjdk-8-jre
+FROM openjdk:8
 
 # Environment variables.
 ENV CENTRALDOGMA_HOME "/opt/centraldogma"
 ENV CENTRALDOGMA_OPTS "-nodetach"
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
```
Comment (hyangtack) [inline dist/Dockerfile]: Use the default java home where is `/docker-java-home`.

**Comment by hyangtack** [comment]
This PR is related to #267 and #268

File: dist/Dockerfile [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 ENV JAVA_OPTS "$CENTRALDOGMA_OPTS"
 
 # Install CentralDogma binaries and configurations.
-RUN mkdir -p                    \
-  "$CENTRALDOGMA_HOME"/bin      \
-  "$CENTRALDOGMA_HOME"/conf     \
-  "$CENTRALDOGMA_HOME"/lib      \
-  "$CENTRALDOGMA_HOME"/licenses \
+RUN mkdir -p                      \
+  "$CENTRALDOGMA_HOME"/bin        \
```
Comment (minwoox) [inline dist/Dockerfile]: Because the option is -p, perhaps we don't need this line?

Comment (hyangtack) [reply dist/Dockerfile]: Yes, right. But I wanted to clearly specify the directories to be created.

**Pull request merged**